build_image.sh

```diff
@@ -150,7 +150,15 @@ elif [ "$IMAGE_TYPE" = "aboot" ]; then
     if [ "$SONIC_ENABLE_IMAGE_SIGNATURE" = "y" ]; then
         TARGET_CA_CERT="$TARGET_PATH/ca.cert"
         rm -f "$TARGET_CA_CERT"
-        [ -f "$CA_CERT" ] && cp "$CA_CERT" "$TARGET_CA_CERT"
+
+        # If the ca certificate does not exist, the test certificate will be used to sign the image
+        if [ ! -f "$CA_CERT" ]; then
+            TEST_CERT_PATH=files/image_config/secureboot/test-certs
+            CA_CERT="${TEST_CERT_PATH}/ca.cert"
+            SIGNING_KEY="${TEST_CERT_PATH}/signing.key"
+            SIGNING_CERT="${TEST_CERT_PATH}/signing.cert"
+        fi
+        cp "$CA_CERT" "$TARGET_CA_CERT"
```

Comment:
Should it be part of another PR?

Reply:
I see. Thanks

Reply:
It should be caused by force push in master branch, the cert had been removed.
@qiluo-msft, the change should not be correct, right?
ActionItem: If your PR contains that commit, then your PR will be impacted. You will need to re-adapt to the revised the git history after the commit is removed.

Reply:
Please refer https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!topic/sonicproject/aV8L81MCnpU

> If your PR contains that commit, then your PR will be impacted. You will need to re-adapt to the revised the git history after the commit is removed.